lib/Switch.js

```diff
@@ -112,7 +115,7 @@ export class Switch extends Component {
 
     if (changeValueImmediately) {
       this.animateSwitch(!propValue);
-      onValueChange(propValue);
+      onValueChange(!propValue);
```

Comment:
Why you added `!` ?

Reply:
In my testing, the master code only ever sent ‘true’ and never turned the switch off. It seemed like a bug that it would send ‘propValue’ because then it would never actually change.